WordPress/src/main/java/org/wordpress/android/ui/posts/PostUtils.java

```diff
@@ -266,7 +266,10 @@ private static String makeExcerpt(String description) {
             return null;
         }
 
-        String s = HtmlUtils.fastStripHtml(removeWPGallery(description));
+        // Remove Gutenberg blocks.
```

Comment:
```suggestion
        // Remove certain Gutenberg blocks that would display markup, rather than their associated media, in the excerpt.
```

A small thing but, if I was reading the code without the context, I wonder if I'd be confused why these two specific blocks were being removed. What do you think about adding more reasoning for removing the blocks in the comment?

Reply:
Good idea, I'll expand the comment to make that part clearer 👍 .